### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by clbarnes** [svenpeter42/fastfilters#23]

**Comment by svenpeter42** [comment]
Yup, the code could also be cleaned up and have more reasonable test cases.
Unfortunately I don’t work in academia anymore and won’t have enough time to tackle this - sorry!
I’ll happily accept PRs though.
